cfme/tests/cli/test_appliance_console.py

```diff
@@ -975,7 +978,63 @@ def test_appliance_console_network_conf():
             9.
             10. Check IP and hostname values matches in Summary Information.
     """
-    pass
+    appliance = temp_appliance_preconfig_modscope
+    command_set = ("ap", RETURN)
+    old_result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    logger.info('"ap" command output before test run:%s' % old_result)
+    first_screen_old = old_result[0].split("\n")
+    ipv4 = re.match(IPv4_REGEX, "".join([i for i in first_screen_old
+                                         if "IPv4 Address:" in i]).strip()).group(2)
+    soft_assert(ipv4)
+    command_set = ("ap", RETURN, "1", "2", ipv4, RETURN, RETURN, RETURN, RETURN, RETURN, "Y")
+    output = appliance.appliance_console.run_commands(command_set, timeout=40, output=True)
+    soft_assert("failed" not in output[-1])
+
+    ipv6 = re.match(IPv6_REGEX, "".join([i for i in first_screen_old
+                                         if "IPv6 Address:" in i]).strip()).group(2)
+    soft_assert(ipv6)
+    command_set = ("ap", RETURN, "1", "3", ipv6, RETURN, RETURN, RETURN, RETURN, RETURN, "Y")
+    output = appliance.appliance_console.run_commands(command_set, timeout=40, output=True)
+    soft_assert("failed" not in output[-1])
+
+    hostname = fauxfactory.gen_alphanumeric(start="test", length=20).lower()
+    command_set = ("ap", RETURN, "1", "5", hostname, RETURN)
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    appliance.evmserverd.restart()
+
+    command_set = ("ap", RETURN)
+    new_result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    logger.info('"ap" command output after restart EVM service:%s' % new_result)
+    first_screen_new = new_result[0].split("\n")
+
+    command_set = ("ip a")
+    ip_a_ret = appliance.ssh_client.run_command(command_set)
+    soft_assert(ip_a_ret.success)
+    ip_a_output = ip_a_ret.output.split("\n")
+    logger.info('"ip a" command output:%s' % ip_a_output)
+
+    soft_assert([i for i in first_screen_new if "IPv4 Address:" in i and ipv4 in i], (
```

Comment:
Can you flatten this a bit to make it more readable?

`soft_assert([True for line in first_screen_new if f'IPv4 Address: {ipv4}' in line]))`

Reply:
A: I think here we can not make like `f'IPv4 Address: {ipv4}'` because we don't know how many spaces will be in between `IPv4 Address` text and actual ip, please correct me if I am wrong

Reply:
We can with a regex match instead of using `in` ... checking your updates.

Reply:
Done